ci/render_pipelines.py

```diff
@@ -212,33 +212,40 @@ def mk_pipeline_promote_task(
 
 def mk_pipeline_notify_task(
     previous_tasks: typing.List[str],
-    gardenlinux_flavour: GardenlinuxFlavour,
-    all_tasks: typing.Sequence[tkn.model.Task]
+    all_tasks: typing.Sequence[tkn.model.Task],
+    build_tasks: typing.Sequence[tkn.model.Task],
+    platform_set: typing.Set[str] = set(),
```

Comment:
it is a bad practise to use modifyable values as default values. reason: if within the function, the argument were to be modified, those modifications would persist between different calls that do not explicitly pass the parameter.

Reply:
Changed